src/persistReducers.js

```diff
@@ -5,12 +5,14 @@ import {
   PURGE,
   REHYDRATE,
   DEFAULT_VERSION,
+  getStoredState,
+  createPersistoid
 } from 'redux-persist'
 import { Map } from 'immutable';
 import purgeStoredState from 'redux-persist/lib/purgeStoredState'
-import createPersistoid from './createPersistoid'
 import autoMergeLevel1 from './stateReconciler/autoMergeLevel1';
-import defaultGetStoredState from './getStoredState'
+
+const defaultGetStoredState = getStoredState;
```

Comment:
we try to get stored state saved by redux-persist persistoid so we can use defaul getStoredState from redux-persist